schemas/2015-10-01/Microsoft.DataFactory.json

```diff
@@ -141,7 +141,7 @@
           ]
         },
         "type": {
-          "enum": [ "Microsoft.DataFactory/datafactories/pipelines" ]
+          "enum": [ "Microsoft.DataFactory/datafactories/datapipelines" ]

```

Comment:
type should be exactly same with the word in service's manifest, including casing.